Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SignInViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+import UIKit
+
+protocol Validation {
+    func isValidCredentials(username: String, password: String) -> Bool
+}
+
+class SignInViewController: UIViewController, Validation {
+    @IBOutlet weak private var txtUsername: UITextField!
+    @IBOutlet weak private var txtPassword: UITextField!
+    @IBOutlet weak private var btnSignIn: UIButton!
+    @IBOutlet weak private var btnRegister: UIButton!
+    var isLoggedIn = false
```

Comment:
All these vars and lets can be private 😄

Reply:
Noted 👍🏻